com.unity.render-pipelines.universal/Shaders/Utils/Blit.shader

```diff
@@ -30,9 +36,44 @@ Shader "Hidden/Universal Render Pipeline/Blit"
 
                 half4 col = SAMPLE_TEXTURE2D_X(_SourceTex, sampler_SourceTex, input.uv);
 
-             #ifdef _LINEAR_TO_SRGB_CONVERSION
+             	#ifdef _LINEAR_TO_SRGB_CONVERSION
                 col = LinearToSRGB(col);
-             #endif
+             	#endif
+
+                #ifdef _DEBUG_HIGHLIGHT_NAN_INF_NEGATIVE_PIXELS
```

Comment:
Maybe this should be just a separate DebugBlit shader? or could this be merged with FullscreenDebug?

Reply:
I'll look into that - it's one of the few shaders I haven't touched, so it will still be however it was implemented at Hackweek.